test/integration/persistentvolumes/persistent_volumes_test.go

```diff
@@ -27,8 +27,6 @@ import (
 	"testing"
 	"time"
 
-	"github.com/golang/glog"
```

Comment:
The same comment for openshift_test.go - shouldn't the package be openshift there?